hilo.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'casino.rb'
+
+  def game_prompt
+    print "Let's play some high low!"
+    puts "
+    1) Hit me!
+    2) Yo im outta here
+    "
+    choice = gets.strip.to_i
+
+    case choice
+    when 1
+      loop
+    when 2
+      print "Later bro!"
+      exit
```

Comment:
remove the exit here. Once the class it done it will end and go back to the casino. Don't want to force an exit.